### DISCUSSION TITLE
Is there a way to clear a sensor cursor?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8557] [Q&A]
is there a way to clear a sensor cursor?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1645224489108039?thread_ts=1645224489.108039&cid=C01U954MEER).*

**Comment by yuhan** [comment]
You can clear the cursor from the UI on the individual sensor's page.

Or, you can do it through CLI: `dagster sensor cursor --help` 
apidoc: https://docs.dagster.io/_apidocs/cli#dagster-sensor
